src/Data/Sequences.hs

```diff
@@ -1198,6 +1244,38 @@ instance VS.Storable a => IsSequence (VS.Vector a) where
 
 -- | A typeclass for sequences whose elements have the 'Eq' typeclass
 class (MonoFoldableEq seq, IsSequence seq, Eq (Element seq)) => EqSequence seq where
+
+    -- | @'splitElem'@ splits a sequence into components delimited by separator
+    -- element. It's equivalent to 'splitWhen' with equality predicate:
+    --
+    -- > splitElem sep === splitWhen (== sep)
+    splitElem :: Element seq -> seq -> [seq]
+    splitElem x = splitWhen (== x)
+
+    -- | @'splitSeq'@ splits a sequence into components delimited by
+    -- separator subsequence. 'splitSeq' is the right inverse of 'intercalate':
+    --
+    -- > intercalate x . splitSeq x === id
+    --
+    -- 'splitElem' can be considered a special case of 'splitSeq'
+    --
+    -- > 'splitSeq' (singleton sep) === 'splitElem sep'
```

Comment:
You cannot use `''` with `>`. It produces invalid documentation. You must wrap the statement like:
```
@
'splitSeq' (singleton sep) === 'splitElem' sep
@
```